### PR TITLE
taxonomy: Added 'Refined palmolein oil'

### PR DESCRIPTION
**Pull request by hakuamesan**
### What
Added 'Refined palmolein oil' as a synonym of "refined palmolein"

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -14611,6 +14611,18 @@ wikidata:en:Q57312740
 description:en:The olein (liquid) fraction of palm oil.
 # 328 in 9 @2021-09-09
 
+<en:palm oil
+en:refined palmolein oil
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:14615]: I think the main name should be "en:refined palm olein", as we have an entry "en:palm olein" already.

So "en:refined palm olein" should be under "<en:palm olein".

I'm guessing also that "palm olein oil" is the exact same thing as "palm olein" (I could be wrong).

In which case we should also add "palm olein oil" as a synonym of "palm olein".

**Comment by stephanegigandet** [comment]
Hi @hakuamesan 

I'm gathering the feedback from Slack, here is what we should do:

- "en:palm olein oil" should be a new entry (under "<en:palm oil" and "<en:palm olein".
- en:refined should be added to ingredients_processing.txt (that way we don't have to add "refined palm olein oil")

**Comment by alexgarel** [comment]
@stephanegigandet should we close it ? As https://github.com/openfoodfacts/openfoodfacts-server/pull/8253 is merged.

**Comment by stephanegigandet** [comment]
This change has been made in another PR: https://github.com/openfoodfacts/openfoodfacts-server/pull/8253

Thank you @hakuamesan for bringing this up!

**Pull request closed without merging**